### PR TITLE
Docs and lint

### PR DESCRIPTION
**Pull request by mischnic**
~Take a look at index.js. Have we found a bug in clang-format?~ I missed a comma and clang-format didn't complain.

### CODE REVIEW

File: docs/date-time-picker.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <a name="UiDateTimePicker"></a>
 
 ## UiDateTimePicker ⇐ <code>DateTimePickerBase</code>
-A widgets to edit dates and times.
+A widgets to set a date and time of day.
```
Comment (parro-it) [inline docs/date-time-picker.md]: Why "of a day"? Seems strange...

Comment (mischnic) [reply docs/date-time-picker.md]: To make the distinction clearer between "date" and "time". (First changed this in time-picker and then here for constistency).

Can change it back if you like.

Comment (parro-it) [reply docs/date-time-picker.md]: > Can change it back if you like.

I'm unsure... leave it as it is by now (I mean let's keep "of day")

Comment (mischnic) [reply docs/date-time-picker.md]: Too late. (I'm no native speaker either).

**Comment by parro-it** [comment]
Thank you!

**Pull request merged**